### PR TITLE
wait for certs to be present before installing

### PR DESCRIPTION
**Pull request by deads2k**
for https://bugzilla.redhat.com/show_bug.cgi?id=1702104

This waits for unrevisioned certs to be present before continuing.

/hold

hold for proof and unit tests

### CODE REVIEW

**Comment by deads2k** [comment]
proof pull https://github.com/openshift/cluster-kube-apiserver-operator/pull/431

File: pkg/operator/staticpod/controller/installer/installer_controller_test.go [hunk 1]
```diff
@@ -1320,3 +1324,9 @@ func TestSetConditions(t *testing.T) {
 	}
 
 }
+
+func TestEnsureCert(t *testing.T) {
```
Comment (mfojtik) [inline pkg/operator/staticpod/controller/installer/installer_controller_test.go]: nit: implement :-)

Comment (deads2k) [reply pkg/operator/staticpod/controller/installer/installer_controller_test.go]: > nit: implement :-)

hence the hold

File: pkg/operator/staticpod/controller/installer/installer_controller.go [hunk 2]
```diff
@@ -711,6 +751,10 @@ func (c InstallerController) sync() error {
 		return nil
 	}
 
+	if err := c.ensureCerts(); err != nil {
+		return err
```
Comment (mfojtik) [inline pkg/operator/staticpod/controller/installer/installer_controller.go]: can we log the error here? (V(4)) ?

Comment (mfojtik) [reply pkg/operator/staticpod/controller/installer/installer_controller.go]: iow. "why my installer pods are not starting"

Comment (deads2k) [reply pkg/operator/staticpod/controller/installer/installer_controller.go]: > iow. "why my installer pods are not starting"

level above logs

**Pull request merged**